app/src/main/java/com/lambdasoup/quickfit/Constants.java

```diff
@@ -21,10 +21,13 @@
     public static final String NOTIFICATION_CHANNEL_ID_PLAY_INTERACTION = "play_interaction";
     public static final int NOTIFICATION_ALARM = 1;
     public static final String NOTIFICATION_CHANNEL_ID_ALARM = "alarm";
+    public static final int NOTIFICATION_ALARM_BG_IO_WORK = 2;
```

Comment:
I must say I am not a fan of "constant collection files"

Reply:
Me neither, but for exactly this kind of purpose, I'm undecided - on the one hand, keeping the constants as local and private as possible is good. On the other hand, precisely for stuff like notification ids and pending intent ids, there's a clear benefit to having a single list for those per application, because they should be unique per application.

Here, I decided to not blow up this PR with even more changes and kick this decision down the road.